css/styles.css

```
@@ -589,6 +589,19 @@ input:disabled + .slider {
   text-align: left;
 }
 
+/* Styles for Right Alignment of Select */ 
```

There's a trailing white space at the end of this line.
```suggestion
/* Styles for Right Alignment of Select */
```